exchanges/bitstamp/bitstamp_websocket.go

```diff
@@ -309,6 +309,7 @@ func (b *Bitstamp) seedOrderBook(ctx context.Context) error {
 			VerifyOrderbook: b.CanVerifyOrderbook,
 			Bids:            make(orderbook.Items, len(orderbookSeed.Bids)),
 			Asks:            make(orderbook.Items, len(orderbookSeed.Asks)),
+			LastUpdated:     time.Now(),
```

Comment:
Just noticed this, can we use the orderbookSeed timestamp so its closer to the matching engine time. and also in `wsUpdateOrderbook` we can use `time.UnixMicro(update.Microtimestamp)` for the `lastupdate` field.

Reply:
Agreed. On it.